ci/downgrade/phpstan.neon

```diff
@@ -0,0 +1,37 @@
+parameters:
+    level: 0
+
+    paths:
+        - ../../src
+        - ../../rules
+        - ../../packages
+        - ../../utils
+
+    reportUnmatchedIgnoredErrors: false
+
+    scanDirectories:
+        - ../../stubs
+
+    excludes_analyse:
+        # List of all excluded elements for Rector and its dependencies
+        # from ci/downgrade/configuration.php
+        # Rector:
+        - */tests/*
+        - ../../ci/*
+        # - ../../stubs/* # Even if not downgraded, they are needed to run PHPStan
+        - ../../packages/rector-generator/templates/*
+        - ../../packages/doctrine-annotation-generated/src/*
+        # Dependencies:
+        - ../../vendor/symfony/cache/DoctrineProvider.php
+        - ../../vendor/symfony/cache/Messenger/EarlyExpirationHandler.php
+        - ../../vendor/symfony/http-kernel/HttpKernelBrowser.php
+        - ../../vendor/symfony/string/Slugger/AsciiSlugger.php
+        - ../../vendor/symfony/cache/Adapter/CouchbaseBucketAdapter.php
+
+        # broken in PHPStan https://github.com/rectorphp/rector/runs/1305002460#step:5:56
+        - ../../utils/phpstan-extensions/*
+        - ../../packages/testing/src/PHPUnit/*.php
+
+    ignoreErrors:
+        # trait in trait call
+        - '#Call to an undefined method Rector\\PostRector\\Rector\\AbstractPostRector\:\:isObjectType\(\)#'
```

Comment:
This should be autogenerated from `ci/downgrade/configuration.php`

Reply:
What do you mean? Have you already done that somewhere else, so I can reproduce it?

I copied that error from the [original `phpstan.neon`](https://github.com/rectorphp/rector/blob/master/phpstan.neon#L291) (all the other errors do not happen in this context, so could be removed)